src/dummy/d2d1.h

```diff
@@ -1006,4 +1037,70 @@ struct dummy_ID2D1SolidColorBrush_tag {
 #define dummy_ID2D1SolidColorBrush_SetColor(self,a)             (self)->vtbl->SetColor(self,a)
 
 
+/*********************************************
+ ***  Interface ID2D1LinearGradientBrush   ***
+ *********************************************/
+
+typedef struct dummy_ID2D1LinearGradientBrushVtbl_tag dummy_ID2D1LinearGradientBrushVtbl;
+struct dummy_ID2D1LinearGradientBrushVtbl_tag {
+    /* IUnknown methods */
+    STDMETHOD(QueryInterface)(dummy_ID2D1LinearGradientBrush*, REFIID, void**);
+    STDMETHOD_(ULONG, AddRef)(dummy_ID2D1LinearGradientBrush*);
+    STDMETHOD_(ULONG, Release)(dummy_ID2D1LinearGradientBrush*);
+
+    /* ID2D1Resource methods */
+    STDMETHOD(dummy_GetFactory)(void);
+
+    /* ID2D1Brush methods */
+    STDMETHOD(dummy_SetOpacity)(void);
+    STDMETHOD(dummy_SetTransform)(void);
+    STDMETHOD(dummy_GetOpacity)(void);
+    STDMETHOD(dummy_GetTransform)(void);
+
+    /* ID2D1LinearGradientBrushBrush methods */
+    STDMETHOD(dummy_GetEndPoint)(void);
+    STDMETHOD(dummy_GetGradientStopCollection)(void);
+    STDMETHOD(dummy_GetStartPoint)(void);
+    STDMETHOD(dummy_SetEndPoint)(void);
+    STDMETHOD(dummy_SetStartPoint)(void);
+};
+
+struct dummy_ID2D1LinearGradientBrush_tag {
+    dummy_ID2D1LinearGradientBrushVtbl* vtbl;
+};
+
+#define dummy_ID2D1LinearGradientBrush_QueryInterface(self,a,b)     (self)->vtbl->QueryInterface(self,a,b)
+#define dummy_ID2D1LinearGradientBrush_AddRef(self)                 (self)->vtbl->AddRef(self)
+#define dummy_ID2D1LinearGradientBrush_Release(self)                (self)->vtbl->Release(self)
+
+
+/************************************************
+ ***  Interface ID2D1GradientStopCollection   ***
+ ************************************************/
+
+typedef struct dummy_ID2D1GradientStopCollectionVtbl_tag dummy_ID2D1GradientStopCollectionVtbl;
+struct dummy_ID2D1GradientStopCollectionVtbl_tag {
+    /* IUnknown methods */
+    STDMETHOD(QueryInterface)(dummy_ID2D1GradientStopCollection*, REFIID, void**);
+    STDMETHOD_(ULONG, AddRef)(dummy_ID2D1GradientStopCollection*);
+    STDMETHOD_(ULONG, Release)(dummy_ID2D1GradientStopCollection*);
+
+    /* ID2D1Resource methods */
+    STDMETHOD(dummy_GetFactory)(void);
+
+    /* ID2D1GradientStopCollection methods */
+    STDMETHOD(dummy_GetColorInterpolationGamma)(void);
+    STDMETHOD(dummy_GetExtendMode)(void);
+    STDMETHOD(dummy_GetGradientStopCount)(void);
+    STDMETHOD(dummy_GetGradientStops)(void);
+};
```

Comment:
Ditto.

Reply:
Info is from here https://docs.microsoft.com/en-us/windows/desktop/api/d2d1/nn-d2d1-id2d1gradientstopcollection